packages/contracts-bedrock/scripts/upgrades/holocene/.env.example

```diff
@@ -0,0 +1,63 @@
+##############################################
+#               ↓  Required  ↓               #
+##############################################
+
+# Etherscan API key used to verify contract bytecode
+ETHERSCAN_API_KEY=
+
+# RPC URL for the L1 network that matches $NETWORK
+ETH_RPC_URL=
+
+# Private key used to deploy the new contracts for this upgrade
+PRIVATE_KEY=
+
+# Path to the deploy config JSON file
+DEPLOY_CONFIG_PATH=
+
+# Path to the folder where output artifacts will be stored
+OUTPUT_FOLDER_PATH=
+
+# Address of the deployed `SystemConfig` implementation for Holocene.
```

Comment:
Can we get these values automatically? We should be able to pull them from the release itself.

Reply:
We cannot. These addresses are not going to be contained within the local deployment - someone else would have deployed these for the user.

And until they are deployed, they won't be present anywhere. We would need to first do the deployment of these contracts for our own chain, then add them to superchain registry, then use something like your [`fetch_standard_addresses`](https://github.com/ethereum-optimism/optimism/blob/perm/op-contracts/v1.6.0/packages/contracts-bedrock/scripts/upgrades/v1.3.0/v1.6.0-permissioned/scripts/common.sh#L95) function.

Reply:
Updated to include this. Just means we have to deploy and get the std config addresses into the registry, before distributing this image to folks.